src/NetscriptFunctions/Formulas.ts

```diff
@@ -165,6 +165,7 @@ export function NetscriptFormulas(): InternalAPI<IFormulas> {
         checkFormulasAccess(ctx);
         return calculatePercentMoneyHacked(server, person);
       },
+      // TODO 2.3: Remove growPercent, add growMultiplier function. Much better name given the output.
```

Comment:
Although that is a better idea, I don't think the churn from breaking scripts is worth the slightly better name.

Reply:
Yeah still not sure. It also has a wacky signature compared to the other hacking formulas so I'm leaning towards changing the name, the removedFunction dialog can let the player know the replacement function to use and know about the param order swap.